site/content/contribute/more-info/server/style-guide.md

```diff
@@ -159,7 +159,27 @@ func OtherFunction() {
 }
 ```
 
-### Log levels
+### Logging
+
+Log messages should be annotated with contextual information in the form of key-value pairs to make it easier to identify the context they originated from. The keys should use snake_case.
```

Comment:
nit: Maybe we can add a sentence like "Refer to the corresponding model object and struct tags for json" so that readers can have a reference.

Reply:
Good suggestion :+1: I've added a naming suggestion via be04bb1.